.lando.yml

```diff
@@ -0,0 +1,9 @@
+name: usher
```

Comment:
@apotek Since we are including Lando config, do you mind if I push one up for DDEV?

Reply:
> @apotek Since we are including Lando config, do you mind if I push one up for DDEV?

No problem at all! I guess I should have added this in a separate branch, but it was just for helping me develop so I left it here. But sure, a ddev would be helpful too.